### ISSUE TITLE
putScript failed when pushing hello-wasm-worker

### ISSUE THREAD

**Issue by nissy-dev** [skymethod/denoflare#21]
I try to push [`hello-wasm-worker`](https://github.com/skymethod/denoflare/tree/master/examples/hello-wasm-worker), but I couldn't with the error putScript raised.
I seem `importWasm` function cause this error, but I couldn't understand the details...

### Error log
```
bundling test-wasm-worker into bundle.js...
bundle finished (process) in 361ms
computed bindings in 1ms
putting module-based worker test-wasm-worker... (38.2kb) (6.3kb compressed)
error: Uncaught (in promise) Error: putScript failed: status=400, errors=10021 Uncaught Error: Some functionality, such as asynchronous I/O, timeouts, and generating random values, can only be performed while handling a request.
  at line 0

    {
      code: 10021,
      message: "Uncaught Error: Some functionality, such as asynchronous I/O, timeouts, and generating random values..."
    }
    at execute (https://raw.githubusercontent.com/skymethod/denoflare/v0.5.3/common/cloudflare_api.ts:1044:15)
    at async putScript (https://raw.githubusercontent.com/skymethod/denoflare/v0.5.3/common/cloudflare_api.ts:96:13)
    at async buildAndPutScript (https://raw.githubusercontent.com/skymethod/denoflare/v0.5.3/cli/cli_push.ts:83:9)
    at async Object.push [as handler] (https://raw.githubusercontent.com/skymethod/denoflare/v0.5.3/cli/cli_push.ts:117:5)
    at async CliCommand.routeSubcommand (https://raw.githubusercontent.com/skymethod/denoflare/v0.5.3/cli/cli_command.ts:104:13)
    at async https://raw.githubusercontent.com/skymethod/denoflare/v0.5.3/cli/cli.ts:33:5
```

### Deploy command
```sh
denoflare push test-wasm-worker --config .denoflare.jsonc --account-id $CLOUDFLARE_ACCOUNT_ID --api-token $CLOUDFLARE_API_TOKEN
```
### .denoflare.jsonc
```json
{
    // This jsonc file supports comments and trailing commas!

    // optional schema to get auto-completions when editing this file in vscode, etc
    "$schema": "https://raw.githubusercontent.com/skymethod/denoflare/v0.5.3/common/config.schema.json",

    // define script configurations by name, and their associated bindings and options
    // you can then simply refer to them by name in your denoflare commands
    // note the same script source can be specified more than once with different configurations (local vs prod etc)
    "scripts": {
        "test-wasm-worker": {
            "path": "hello.ts",
            "localPort": 3002
        }
    }
}

```

### Environments

- Mac OS Monterey (M1)
- deno 1.23.4
- denoflare 0.5.3

**Comment by johnspurlock-skymethod** [comment]
Cloudflare does a quick static analysis on the script on upload.  It will give you these errors if you try to do IO or Date.now() etc in the global scope.

Any differences between your local copy and the example?

Note you can try pushing the example as it is:
```
$ denoflare push --name test-wasm-worker https://raw.githubusercontent.com/skymethod/denoflare/master/examples/hello-wasm-worker/hello.ts
```

Or by replacing `"path": "hello.ts",` with `"path": "https://raw.githubusercontent.com/skymethod/denoflare/master/examples/hello-wasm-worker/hello.ts",` in your config.

**Comment by nissy-dev** [comment]
@johnspurlock-skymethod 
Thank you for your comments! 

> Any differences between your local copy and the example?

There were differences because I made a mistake about my editor setting. If I remove them, I can push.
The differences was that Prettier formmated `hello.ts`. I don't understand why the differences effect whether I can push or not.

### differences
```
diff --git a/examples/hello-wasm-worker/hello.ts b/examples/hello-wasm-worker/hello.ts
index b148843..b2607dd 100644
--- a/examples/hello-wasm-worker/hello.ts
+++ b/examples/hello-wasm-worker/hello.ts
@@ -1,24 +1,28 @@
-import { importWasm } from './deps.ts';
-import { callSub } from './sub/sub.ts'; // also works with relative module specifiers within relative imports
-const module = await importWasm(import.meta.url, './hello.wasm'); // rewritten to: import module from './hello.wasm';
+import { importWasm } from "./deps.ts";
+import { callSub } from "./sub/sub.ts"; // also works with relative module specifiers within relative imports
+const module = await importWasm(import.meta.url, "./hello.wasm"); // rewritten to: import module from './hello.wasm';

 export default {
+  fetch(): Response {
+    try {
+      // call hello.wasm
+      const instance = new WebAssembly.Instance(module);
+      const result = (instance.exports.main as CallableFunction)();

-    fetch(): Response {
-        try {
-            // call hello.wasm
-            const instance = new WebAssembly.Instance(module);
-            const result = (instance.exports.main as CallableFunction)();
+      // call hello2.wasm
+      const subresult = callSub();

-            // call hello2.wasm
-            const subresult = callSub();
-
-            const html = `<h3>Result from hello.wasm call: ${result}</h3><h3>Result from hello2.wasm call: ${subresult}</h3>`;
-            return new Response(html, { status: 200, headers: { 'Content-Type': 'text/html; charset=utf-8' } });
-        } catch (e) {
-            const html = `${e}`;
-            return new Response(html, { status: 500, headers: { 'Content-Type': 'text/html; charset=utf-8' } });
-        }
+      const html = `<h3>Result from hello.wasm call: ${result}</h3><h3>Result from hello2.wasm call: ${subresult}</h3>`;
+      return new Response(html, {
+        status: 200,
+        headers: { "Content-Type": "text/html; charset=utf-8" },
+      });
+    } catch (e) {
+      const html = `${e}`;
+      return new Response(html, {
+        status: 500,
+        headers: { "Content-Type": "text/html; charset=utf-8" },
+      });
     }
-
+  },
 };
```

### formated code

```ts
import { importWasm } from "./deps.ts";
import { callSub } from "./sub/sub.ts"; // also works with relative module specifiers within relative imports
const module = await importWasm(import.meta.url, "./hello.wasm"); // rewritten to: import module from './hello.wasm';

export default {
  fetch(): Response {
    try {
      // call hello.wasm
      const instance = new WebAssembly.Instance(module);
      const result = (instance.exports.main as CallableFunction)();

      // call hello2.wasm
      const subresult = callSub();

      const html = `<h3>Result from hello.wasm call: ${result}</h3><h3>Result from hello2.wasm call: ${subresult}</h3>`;
      return new Response(html, {
        status: 200,
        headers: { "Content-Type": "text/html; charset=utf-8" },
      });
    } catch (e) {
      const html = `${e}`;
      return new Response(html, {
        status: 500,
        headers: { "Content-Type": "text/html; charset=utf-8" },
      });
    }
  },
};

```

**Comment by nissy-dev** [comment]
@johnspurlock-skymethod 
I could push the script to Cloudflare Worker with your modification  :+1: 
I appreciate your quick modification and clear explanation. Thank you!!

**Comment by johnspurlock-skymethod** [comment]
This change is included in release [v0.5.4](https://github.com/skymethod/denoflare/releases/tag/v0.5.4)
